### PR TITLE
Fix removing old nodes after restore if rke2-server already started

### PR DESCRIPTION
**Pull request by glennpratt**
# Description

If rke2-server happened to start on the first node before the node removal section, nodes would not be removed.

This was because the removal was in `first_server.yml` which is gated by `active_server is not defined`.

https://github.com/lablabs/ansible-role-rke2/blob/main/tasks/main.yml#L46-L49

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Small minor change not affecting the Ansible Role code (GitHub Actions Workflow, Documentation etc.)

## How Has This Been Tested?

TBD

### CODE REVIEW

**Comment by MonolithProjects** [comment]
Hi @glennpratt, thanks for this PR. Please resolve the conflict.

File: tasks/first_server_restore.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+- name: Restore etcd - get all nodes
+  ansible.builtin.shell: |
+    set -o pipefail
```
Comment (MonolithProjects) [inline tasks/first_server_restore.yml:3]: pipefail is not needed here

**Review by MonolithProjects — CHANGES_REQUESTED** [review]
Hi @glennpratt, please fix the pipefail. Everything else looks fine. Thanks

File: tasks/first_server_restore.yml (hunk 1 above)
Comment (glennpratt) [reply tasks/first_server_restore.yml:3]: Oh, good catch, copied over the merge conflict change without thinking.

**Review by MonolithProjects — APPROVED** [review]
LGTM. Thanks

**Pull request merged**